### PR TITLE
refactor: refactor showing result graphs and fix centralized flow

### PR DESCRIPTION
**Pull request by Adibov**

### CODE REVIEW

File: app/config/config.py
Comment (negaranabestani) [inline app/config/config.py:1]: this index is set from input args in the input parser. is there any reason doing it in the config file?

File: app/entity/fed_client.py
Comment (negaranabestani) [inline app/entity/fed_client.py:1]: why the initialize method is deleted?

File: app/fl_method/fl_method_parser.py
Comment (negaranabestani) [inline app/fl_method/fl_method_parser.py:1]: I think it was better to keep the methods and the ability to support rl splitting.

File: app/model/entity/rl_model/AC.py
Comment (negaranabestani) [inline app/model/entity/rl_model/AC.py:1]: I understand that you wanted to keep things clear but I think this was supposed to be a gama project and the base project features should not  be deleted.

File: app/rl_training/runner/rl_server_runner.py
Comment (negaranabestani) [inline app/rl_training/runner/rl_server_runner.py:1]: i think you can keep the RL files but exclude them from docker image if necessary.

File: app/entity/fed_client.py
Comment (Adibov) [reply app/entity/fed_client.py:1]: Because it resets the model every time and we don't need it.

File: app/config/config.py
Comment (Adibov) [reply app/config/config.py:1]: You're right, I'll remove it from here.

File: app/fl_method/fl_method_parser.py
Comment (Adibov) [reply app/fl_method/fl_method_parser.py:1]: Let's keep the repo as minimal as possible and remove dead codes. If we plan to support the RL splitting, we can add it from the git history.

File: app/model/entity/rl_model/AC.py
Comment (Adibov) [reply app/model/entity/rl_model/AC.py:1]: Having RL codes doesn't mean supporting RL features. Our codes already have diverged from the base project and if we want to support the RL (in the future) we have to copy these files and change them accordingly.

File: app/rl_training/runner/rl_server_runner.py
Comment (Adibov) [reply app/rl_training/runner/rl_server_runner.py:1]: Answered in previous comments.

File: app/entity/fed_client.py
Comment (negaranabestani) [reply app/entity/fed_client.py:1]: I disagree. it doesn't reset the model it initializes the model with new layer config but in the cases that there is no splitting it should not be called. note this and apply changes if needed. thanks.

**Pull request merged**